tools/installers/install_warp-rnnt.sh

```diff
@@ -10,22 +10,9 @@ fi
 
 cuda_ver=$1
 
-if [ $cuda_ver != "10.0" ]; then
-    echo "warp-rnnt was only tested with CUDA_VERSION=10.0. Skipping install."
+if [ $cuda_ver != "10.0" ] || [ $cuda_ver != "10.1" ] || [ $cuda_ver != "10.2" ]; then
+    echo "warp-rnnt was not tested with CUDA_VERSION=$cuda_ver. Skipping install."
     exit 0
 fi
 
-# TODO(kamo): Consider clang case
-# Note: Requires gcc>=5.4 to build extensions with pytorch>=1.0
-if python3 -c 'import torch as t;assert t.__version__[0] >= "1.0"' &> /dev/null; then \
-    python3 -c "from distutils.version import LooseVersion as V;assert V('$(gcc -dumpversion)') >= V('5.4'), 'Requires gcc>=5.4'"; \
-    
-fi
-
-rm -rf warp-rnnt
-git clone https://github.com/1ytic/warp-rnnt
-
-(
-    set -euo pipefail
-    cd warp-rnnt/pytorch_binding && python3 setup.py install
-)
+pip install warp_rnnt
```

Comment:
I would prefer not relying on pypi package. If needed we can quickly fix through PR or our own fork and not wait for pypi package new version.

Reply:
Ok, I'll revert the original installation back